exist-core/src/main/java/org/exist/client/InteractiveClient.java

```diff
@@ -137,6 +138,7 @@ public class InteractiveClient {
         defaultProps.setProperty(USER, USER_DEFAULT);
         defaultProps.setProperty(EDITOR, EDIT_CMD);
         defaultProps.setProperty(INDENT, "true");
+        defaultProps.setProperty(OUTPUT_DOCTYPE, "true");
```

Comment:
Could this break existing applications or was the default to output doctype declarations true by default before?